coralproject/talk
# [CORL-1637] Autohide scrollbar

## What does this PR do?

This PR hides the scroll bar in the Giphy widget when there are no results.

## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

1. Enable always showing scroll bars on OSX (https://heresthethingblog.com/2015/03/10/mac-tip-macs-scroll-bars/)
2. visit a comment thread
3. Click the `+ GIF` button
4. Before entering a gif search, observe that there is no scroll bar between the search box and the Giphy logo
5. Note that the Giphy logo is no longer stretched vertically in Safari or Firefox
